IpmiFeaturePkg/GenericIpmi/Common/GenericIpmi.c

```diff
@@ -301,26 +301,28 @@ IpmiSendCommandInternal (
     break;
   }
 
-  //
-  // Copy data over to the response data buffer.
-  //
-  *ResponseDataSize = DataSize - IPMI_RESPONSE_HEADER_SIZE;
-  CopyMem (
-    ResponseData,
-    IpmiResponse->ResponseData,
-    *ResponseDataSize
-    );
-
-  //
-  // Add completion code in response data to meet the requirement of IPMI spec 2.0
-  //
-  *ResponseDataSize += 1; // Add one byte for Completion Code
-  for (Index = 1; Index < *ResponseDataSize; Index++) {
-    ResponseData[*ResponseDataSize - Index] = ResponseData[*ResponseDataSize - (Index + 1)];
-  }
+  if (ResponseData != NULL && ResponseDataSize != NULL) {

```

Comment:
Need to update the header here and in the .h file to add the `OPTIONAL` note for ResponseData and ResponseDataSize